libs/domain/user/navigation-trigger/navigation-trigger.component.ts

```diff
@@ -0,0 +1,47 @@
+import { AuthService } from '@spryker-oryx/auth';
+import { resolve } from '@spryker-oryx/di';
+import { ContentMixin } from '@spryker-oryx/experience';
+import { RouteType, RouterService } from '@spryker-oryx/router';
+import { LinkService } from '@spryker-oryx/site';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { UserService } from '@spryker-oryx/user';
+import { hydrate, signal } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+
+@hydrate()
+export class UserNavigationTriggerComponent extends ContentMixin(LitElement) {
```

Comment:
UserAuthNavigationComponent? 🤔 
Not sure, but this "navigation trigger" sounds not too clear for me...